.github/workflows/projects_check.yml

```
@@ -75,6 +75,8 @@ jobs:
 
   style_check:
     name: Style
+    needs: changed
+    if: ${{ needs.changed.outputs.editorconfig_files }}
```

```suggestion
    if: ${{ needs.changed.outputs.editorconfig_files == 'true' }}
```

yes you do need that